mw-webapp/src/styles/_variables.scss

```diff
@@ -10,6 +10,7 @@ $link-active: rgb(255, 149, 0);
 
 // Header
 $header-background: rgb(85, 122, 149);
+$header-padding: 0 1em;
```

Comment:
If this padding will be used only in header then need move this var to header.scss. Better skip all styles changes on task #41

Reply:
move all header styles from variables.scss to header.module.scss?